models/iaf_psc_alpha.cpp

```diff
@@ -257,6 +255,8 @@ iaf_psc_alpha::init_buffers_()
 void
 iaf_psc_alpha::calibrate()
 {
+  B_.input_buffer_.resize();
```

Comment:
Why this `resize()` call here? If I am not entirely mistaken, buffer sizes should not change once buffers have been created, i.e., min/max delay cannot change after `Simulate` has been called.

Reply:
I have changes this to call `clear()` here, which calls `resize()`, which only resizes the buffer if needed. This is consistent with the previous code. If `clear()` (and hence `resize()`) is not required here, this should be consistently changed for all neuron models.